test/e2e/.eslintrc.js

```diff
@@ -0,0 +1,181 @@
+// For the most part, this file mirrors the same configuration from the root
+// `.eslintrc.json`, the exceptions being that since Cypress is preconfigured
+// with Mocha & Chai, we need to broadly disable the Jest rules, which is
+// otherwise difficult to do. This file could be made more minimal once all of
+// the Gutenberg-specific rules are migrated to a common WordPress config.
```

Comment:
I wonder if there's a way to override the root config instead of copying it

Reply:
Agreed, that would be the best solution. I would leave it as it is. We are going to tackle it together with @ntwb as part of `@wordpress/packages` soon.

Reply:
The previous iteration of the configuration file is how to override the root config (i.e. ESLint will look in all parent directories until it finds a config with `root: true`). The problem is that by opting into many Jest presets, there are many rules and `env` values we'd need to disable individually.

Reply:
It's more complicated than I would expect.